packages/contracts-sdk/src/interfaces/farcaster/IFarcasterIdRegistryContract.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  BlockchainCommonErrors,
+  EVMAccountAddress,
+  FarcasterUserId,
+  FarcasterIdRegistryContractError,
+} from "@snickerdoodlelabs/objects";
+import { ResultAsync } from "neverthrow";
+
+import { IFarcasterBaseContract } from "@contracts-sdk/interfaces/farcaster/IFarcasterBaseContract.js";
+
+export interface IFarcasterIdRegistryContract
+  extends IFarcasterBaseContract<FarcasterIdRegistryContractError> {
+  /**
+   * @notice Calls the idOf mapping in the registry contract
+   *
+   * @param ownerAddress Address to look up
+   *
+   * @return The farcaster id
+   */
+  idOf(
+    ownerAddress: EVMAccountAddress,
+  ): ResultAsync<
+    FarcasterUserId | null,
+    FarcasterIdRegistryContractError | BlockchainCommonErrors
+  >;
+}
```

Comment:
On all the interfaces, make sure to export the type as well for injection purposes.